.circleci/config.yml

```diff
@@ -188,6 +179,14 @@ jobs:
           name: Run e2e admin page test
           command: |
             python -m scripts.run_e2e_tests --skip-install --skip-build --suite="adminPage" --prod_env
+      - run:
+          name: Run e2e accessibility page test
+          command: |
+            python -m scripts.run_e2e_tests --skip-install --skip-build --suite="accessibility" --prod_env
```

Comment:
is this a new change in this PR? I'm not sure whether we should add much load to the parent build as all other builds are waiting on this. Should we move this to any child node?

Reply:
yes, this section was previously failing on Travis when I moved it there. Do you have section that would be fitting of this?

Reply:
I ultimately pushed the accessibility down, but kept the embedding in this section.